chrome_extension/styles.css

```diff
@@ -704,6 +704,13 @@ header {
     left: 30px!important;
 }
 
+.cp-opacity, .cp-memo {
```

Comment:
would you mind adding a comment stating that these styles are to overwrite the jscolor plugin classes and hide opacity menu from the popup.